paper/writeup.tex

```diff
@@ -109,14 +109,21 @@
     it is an old observation that the ``junction'' between two distinct haplotypes
     produced by recombination is inherited as a Mendelian marker.
     In this paper, we describe how this recombination-mediated information
-    can in many cases be recovered from inference based solely on polymorphic markers,
-    which produces unary nodes along the branches of the marginal genealogical trees.
+    can in many cases be recovered from inference based solely on polymorphic markers.
+    % NSP-22NOV: "unary node" and "ARG" are not defined so I tried to provide better context here
+    In a genealogical context, this information reflects the persistence
+    of ancestral haplotypes across marginal genealogical trees where they
+    are not coalescence events.
+    We show how these non-coalescing haplotypes (``unary nodes'') may be
```

Comment:
I think we should call them something at least a bit different than "unary nodes" at least in the abstract, since "unary node" sounds like a node that is *always* unary.

Reply:
"unary regions" should work I think